examples/allennlp_simple.py

```diff
@@ -0,0 +1,118 @@
+import allennlp
+import allennlp.data
+import allennlp.models
+import allennlp.modules
+import optuna
+import torch
+import uuid
+
+
+DEVICE = -1  # If you want to use GPU, use DEVICE = 0
+
+GLOBE_FILE_PATH = (
```

Comment:
I guess this is a typo for `GLOVE` from the file name.
```suggestion
GLOVE_FILE_PATH = (
```